x/profiles/types/models_app_links_test.go

```diff
@@ -371,28 +382,29 @@ func TestResult_Validate(t *testing.T) {
 }
 
 func TestResult_Failed__Validate(t *testing.T) {
-	usecases := []struct {
+	testCases := []struct {
 		name      string
 		result    *types.Result
 		shouldErr bool
 	}{
 		{
-			name:      "invalid result returns error",
+			name:      "invalid error returns error",
```

Comment:
Could it be named with something like `invalid error result returns error`?

Reply:
@dadamu This test is to make sure that no invalid error string can be provided, thus returns an error. I don't know how to make it easier to understand :joy:

Reply:
😂
How about `invalid error string of result returns error`?

Reply:
Changed :+1: